src/script_rs/src/main.rs

```diff
@@ -0,0 +1,64 @@
+use std::time::Instant;
+use rand::{Rng, SeedableRng};
+use rand_xoshiro::Xoshiro256PlusPlus;
+
+fn dotindex(x: &[f64]) -> f64 {
+    x.iter().enumerate().fold(0.0, |acc, (i, v)| v.mul_add(i as f64, acc))
+}
+
+fn f(n: i32, m: i32) -> f64 {
+    let n = n as usize;
+    let mut y: f64 = 0.0;
+    let mut x: Vec<f64> = Vec::with_capacity(n);
+    x.resize(n, 0.0);
+    let mut rng = Xoshiro256PlusPlus::from_rng(rand::thread_rng()).unwrap();
+
+    for _ in 0..m {
```

Comment:
```suggestion
    for _ in 1..m {
```
Average over `m` trails, not `m+1` (not that it should matter).

If `0..m-1` is more standard rust style, that's okay too.

Reply:
Oops! Just figured out the distinction between `..` and `..=`. This is good as is.